### DISCUSSION TITLE
Cannot running managment port on openvpn server confige

### DISCUSSION THREAD

**Discussion by m4jid-ha** [angristan/openvpn-install#1115] [Help]
Hi all

I want to monitor my openpvn server and i use this solution's "https://github.com/furlongm/openvpn-monitor".
But when i add  "management localhost 7505 " line in end of /etc/openvpn/server.conf file and then restart openvpn server nothing happens.
It mean then i check my run serve ices and listen port with netstat command the 7505 port is unused and free and output of telnet 127.0.0.1 7505 command returned "Connection refused".

thank you for your helps.

**Comment by randshell** [comment]
> But when i add "management localhost 7505 "

Openvpn expects an IP not a hostname. The installation guide at https://github.com/furlongm/openvpn-monitor uses `127.0.0.1` too.
